services/src/main/groovy/org/jfrog/artifactory/client/impl/ArtifactoryImpl.java

```diff
@@ -28,6 +29,8 @@
  */
 public class ArtifactoryImpl implements Artifactory {
 
+	private static final String ORIGINAL_HOST_CONTEXT_PARAM = "original.host.context.param";
```

Comment:
This constant needs to be defined as *public final*, so that it can be also used in the *PreemptiveAuthInterceptor* class. There's no need for the *PreemptiveAuthInterceptor* to also declare the same variable.